connect_ext_ppr/models/deployment.py

```diff
@@ -20,7 +20,7 @@ class Deployment(Model):
 
     PREFIX = 'DPL'
 
-    STATUS = DeploymentStatusChoices
+    STATUSES = DeploymentStatusChoices
```

Comment:
I hope you finally agree between 3 of you @akodelia @jonatrios @Hairash how to declare the enums :-)